### PR TITLE
fix links especially markdown links in sidebar

### LINKED ISSUE #4221: Links in the Fast template sidebar does not style correctly
panel==0.14.2

I'm trying to improve the gapminder example of Panel. One issue is that if you put Markdown with links in the sidebar they don't style well

image

```python
import panel as pn

pn.extension(template="fast")

ACCENT="#00A170"

logo  = """<a href="http://panel.holoviz.org">
           <img src="https://panel.holoviz.org/_static/logo_stacked.png" 
            width=150 height=127 align="left" margin=20px>"""
title = '<h2>Plotting library comparison</h2>'

desc = pn.pane.HTML("""
    The <a href="http://panel.pyviz.org">Panel</a> library from <a href="http://pyviz.org">HoloViz</a> 
    lets you make widget-controlled apps and dashboards from a wide variety of 
    plotting libraries and data types. Here you can try out five different plotting libraries
    controlled by a couple of widgets, for Hans Rosling's 
    <a href="https://demo.bokehplots.com/apps/gapminder">gapminder</a> example.""")

info = pn.Column(
        logo, title, desc
)
info.servable(target="sidebar")

pn.state.template.param.update(
    site="Panel",
    site_url="https://panel.holoviz.org",
    title="Hans Rosling's Gapminder",
    header_background=ACCENT,
    accent_base_color=ACCENT,
)
```

I should have reported and fixed this a long time ago. Major issue with the template.

### PR DESCRIPTION
**Pull request by MarcSkovMadsen**
Closes #4221

### CODE REVIEW

File: panel/template/fast/css/fast_panel_markdown.css [hunk 1]
```diff
@@ -2,11 +2,11 @@
   padding-right: 1em;
   padding-left: 1em;
 }
-.markdown a {
+.main a, .sidebar a {
```
Comment (MarcSkovMadsen) [inline panel/template/fast/css/fast_panel_markdown.css:5]: I could see links in a HTML pane just had the default blue color. It should also have the accent color.

File: panel/template/fast/grid/fast_grid_template.css [hunk 2]
```diff
@@ -124,7 +124,7 @@ img.app-logo {
 .sidenav fast-menu {
   width: 100%;
 }
-.sidenav a {
+.sidenav fast-menu a {
```
Comment (MarcSkovMadsen) [inline panel/template/fast/grid/fast_grid_template.css:127]: This css was originally added to support creating a HTML based expandable menu like the one previously at awesome-panel.org.

**Comment by philippjfr** [comment]
Thanks, was about to make the same changes.

**Review by philippjfr — APPROVED** [review]
Looks good, thank you!

**Pull request merged**